AntiCheat/Lang/en_US.resx

```diff
@@ -121,100 +121,100 @@
     <value>Boss attack spoofing detection</value>
   </data>
   <data name="config_ChatReal" xml:space="preserve">
-    <value>Fake chat message detection</value>
+    <value>Detects when a player attempts to spoof their name as another player's.</value>
   </data>
   <data name="config_DespawnItem" xml:space="preserve">
-    <value>Abnormal item despawn detection</value>
+    <value>Detects when items are unusually destroyed or deleted.</value>
   </data>
   <data name="config_Enemy" xml:space="preserve">
-    <value>Abnormal monster behavior detection</value>
+    <value>Detects when a monster behaves abnormally.</value>
   </data>
   <data name="config_FreeBuy" xml:space="preserve">
-    <value>Free purchases detection</value>
+    <value>Detects players ordering items without spending credits.</value>
   </data>
   <data name="config_Gift" xml:space="preserve">
-    <value>Abnormal gift box interaction detection</value>
+    <value>Detects players having abnormal gift box interactions.</value>
   </data>
   <data name="config_GrabObject" xml:space="preserve">
-    <value>Item pickup distance detection</value>
+    <value>Detects picking up items outside of default grab range.</value>
   </data>
   <data name="config_InfiniteAmmo" xml:space="preserve">
-    <value>Infinite ammo detection</value>
+    <value>Detects when a player has infinite ammo.</value>
   </data>
   <data name="config_Invisibility" xml:space="preserve">
-    <value>Invisible player detection</value>
+    <value>Detects invisible players.</value>
   </data>
   <data name="config_ItemCooldown" xml:space="preserve">
-    <value>Abnormal item interaction cooldown detection</value>
+    <value>Detects when a player uses an item faster than they should.</value>
   </data>
   <data name="config_Jetpack" xml:space="preserve">
-    <value>Abnormal jetpack explosion detection</value>
+    <value>Detects when a player forces a jetpack explosion.</value>
   </data>
   <data name="config_Kick" xml:space="preserve">
-    <value>Kick player when this behavior is detected</value>
+    <value>Kick player when this behavior is detected.</value>
   </data>
   <data name="config_KillEnemy" xml:space="preserve">
-    <value>Enemy kill detection</value>
+    <value>Detects when an enemy is instantly killed.</value>
   </data>
   <data name="config_Landmine" xml:space="preserve">
-    <value>Abnormal landmine triggering detection</value>
+    <value>Detects when a player abnormally detonates a landmine.</value>
   </data>
   <data name="config_Log" xml:space="preserve">
     <value>Show log</value>
   </data>
   <data name="config_Map" xml:space="preserve">
-    <value>Mini-Map mod detection</value>
+    <value>Detects players using a mini-map mod, displaying the radar on the screen at all times.</value>

```

Comment:
remove additional part to keep original meaning

Reply:
I feel like the additional information here is legitimately important. Not every player knows what a mini-map is or what it implies, and the UI has plenty of space for this info. I would like to recommend you keep this.